### PR TITLE
feat [SCRUM-86]: implement classes Vertex3D, Entity, Face, Triangle, Material

### PR DESCRIPTION
**Pull request by lisakorntheuer**
# Description

Added the classes Vertex3D, Entity, Face, Triangle, Material.
Implemented some basic functionalities and structures of those classes (not complete yet).

Open questions:
- Face (& Triangle): Without abstract methods at the moment 
- Code documentation: Right style and complete? 

## ⁉️ Type Of Change

- [x] ⚙️ New feature (non-breaking change which adds functionality).
- [ ] 🪛 Bug fix (non-breaking change).
- [ ] ⚠️ Breaking change (fix of feature that would cause existing functionality to not work as expected).
- [ ] ✏️ This change requires a documentation update.

## 🧪 How Has This Been Tested?

No tests yet.

## ✅ Checklist

- [x] My code follows the style guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have made corresponding changes to the documentation.
- [ ] My changes generate no new warnings.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] New and existing unit tests pass locally with my changes.

### CODE REVIEW

**Comment by adi9113** [comment]
I don't think JavaDoc would be necessary for attributes, but I think it's really helpful though 👍

File: raytracer/src/main/java/rayrangers/raytracer/Material.java [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+     */
+    public Material(String name) {
+        this.name = name;
+    }
+
+    /**
+     * Returns the material name.
+     * @return  material name
+     */
+    private String getName() {
```
Comment (birkenbube) [inline raytracer/src/main/java/rayrangers/raytracer/Material.java]: @lisakorntheuer Shouldn't this be `public`?

File: raytracer/src/main/java/rayrangers/raytracer/Material.java [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+        return name;
+    }
+
+    /**
+     * Sets the material name.
+     * @param name  material name
+     */
+    private void setName(String name) {
```
Comment (birkenbube) [inline raytracer/src/main/java/rayrangers/raytracer/Material.java]: @lisakorntheuer I think this also should be `public`.

**Review by birkenbube — DISMISSED** [review]
Everything resolved

File: raytracer/src/main/java/rayrangers/raytracer/world/Vertex3D.java [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+     */
+    public void setAllCoord(double x1, double x2, double x3) {
+        coordinates = new double[] {x1, x2, x3};
+    }
+
+    /**
+     * Returns the normal vector of the vertex.
+     * @return  normal vector
+     */
+    public Object getNormalVector() {
```
Comment (birkenbube) [inline raytracer/src/main/java/rayrangers/raytracer/world/Vertex3D.java]: Needs to be changed after Vector3D MR is merged

**Review by birkenbube — APPROVED** [review]
Looks good, ready to merge!

**Review by Davidoo98 — APPROVED** [review]
Great work!

**Pull request merged**